examples/llm/requirements.txt

```diff
@@ -1,12 +1,13 @@
 torch==1.13.1
 einops==0.5.0
-mosaicml==0.12.1
-mosaicml-streaming==0.2.3
+mosaicml==0.13.2
+mosaicml-streaming==0.3.0
 flash-attn==0.2.8
 mosaicml-cli>=0.2.32,<1
 # need a newer version of triton
 triton==2.0.0.dev20221202
-transformers==4.25.1
+transformers==4.26.1
 omegaconf==2.2.3
-wandb==0.13.4
+wandb==0.13.6
```

Comment:
any particular reason for changing this one? and did we have some issues upgrading it in the past?

Reply:
I see this more recent one being used by BERT and want to keep them in sync